Add Screenshot#file_size_in_bytes method

marcqualie opened the pull request:
Fixes #1

This converts <String:"0.123 MB"> (#file_size) into <Integer:123000>

marcqualie committed:
Add Screenshot#file_size_in_bytes method

This converts <String:"0.123 MB"> (#file_size) into <Integer:123000>

marcqualie commented:
I put the unit multiplier as 1000 (rather than 1024). Wasn't sure which convention would be preferred/consistent with the rest of the library.

cheshire137 commented on lib/rsteamshot/screenshot.rb:
```diff
@@ -92,6 +92,23 @@ def inspect
+    def file_size_in_bytes
+      unit_multiplier = 1000
+      size, unit = self.file_size.split(' ')
+      bytes = begin
```
I don't think the `begin`/`end` is necessary. You can just have `bytes = case unit.downcase` here.

cheshire137 commented on spec/rsteamshot/screenshot_spec.rb:
```diff
@@ -82,4 +82,31 @@
+
+    it 'converts file_size string in GB into integer of bytes' do
+      expect(screenshot).to receive(:file_size).and_return('0.123 GB')
+      expect(screenshot.file_size_in_bytes).to equal(123000000)
```
Could add underscores to make it more readable: `123_000_000`.

cheshire137 commented on spec/rsteamshot/screenshot_spec.rb:
```diff
@@ -82,4 +82,31 @@
+
+  describe '#file_size_in_bytes' do
+    it 'converts file_size string in B into integer of bytes' do
+      expect(screenshot).to receive(:file_size).and_return('123 B')
```
I usually see `allow` instead of `expect` paired with `to receive`. I guess this works though!

cheshire137 reviewed (approved):
Sorry for the delay reviewing! Thanks for the PR, this looks great! ✨

marcqualie committed:
Tweak file_size_in_bytes based on code review feedback

- Rest of the library uses allow instead of expect for receives
- Improved readability of large numbers
- No need for extra begin statement

marcqualie commented:
Thanks for the feedback. I've updated accordingly.

cheshire137 reviewed (approved):
Thank you!
